pkg/imagestore/imagestore.go

```diff
@@ -84,16 +85,34 @@ const (
 	ImageTypeMinimal = "minimal-iso"
 )
 
-func NewImageStore(ed isoeditor.Editor, dataDir, imageServiceBaseURL string, insecureSkipVerify bool, versions []map[string]string) (ImageStore, error) {
+func NewImageStore(ed isoeditor.Editor, dataDir, imageServiceBaseURL string, insecureSkipVerify bool, versions []map[string]string, additionalCAFile string) (ImageStore, error) {
 	if err := validateVersions(versions); err != nil {
 		return nil, err
 	}
 	transportConfig, ok := http.DefaultTransport.(*http.Transport)
 	if !ok {
 		return nil, fmt.Errorf("expected http.DefaultTransport to be of type *http.Transport")
 	}
+
 	myTransport := transportConfig.Clone()
 	myTransport.TLSClientConfig = &tls.Config{InsecureSkipVerify: insecureSkipVerify} //nolint:gosec // Optionally ignore TLS (G402 error)
+
+	// Add additional TLS certificates (if available) for pulling images.
```

Comment:
The verb `pulling` implies that you're talking about a container registry. This is not accessing a container registry. This is accessing an HTTP server serving OS ISO images over raw HTTP